core/trino-main/src/test/java/io/trino/operator/output/BenchmarkPartitionedOutputOperator.java

```diff
@@ -25,14 +25,14 @@
 import io.trino.jmh.Benchmarks;
```

Comment:
Add to commit message why it the right generation compared to `LocalPartitionGenerator`